lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -29,6 +29,11 @@ click_button:
     selector:
       xpath: //button[contains(.,'<button_text>')]
     op: click
+click_button_text:
```

Comment:
What's the purpose of adding `//button[text()='<button_text>']`?  What's the limitation of `//button[contains(.,'<button_text>')]`

Reply:
The later would find more than one elements when trying to locate substring in one page, for instance, "NoSchedule" is a substring of `PreferNoSchedule`. However I would get two elements when I locate `NoSchedule`